pages/AWorkedExample.html

```diff
@@ -44,19 +44,19 @@ <h4>First Card</h4>
   <p>
     Click on <strong>Add Interaction</strong>.
   </p>
-  <img src="static/images/tutorial/card1AddInteraction.png" class="tutorial-image">
+  <img src="static/images/tutorial/card1AddInteraction.png" class="tutorial-image" alt="Add interaction button">
```

Comment:
Should this be `Add Interaction button` or `'Add Interaction' button`?

Otherwise, sounds like you're adding a button.

Reply:
Makes sense! Will account for it.